src/Microsoft.DotNet.Interactive.Jupyter/IJupyterKernelSpecModule.cs

```diff
@@ -11,5 +12,6 @@ public interface IJupyterKernelSpecModule
     {
         Task<CommandLineResult> InstallKernel(DirectoryInfo sourceDirectory);
         DirectoryInfo GetDefaultKernelSpecDirectory();
+        IReadOnlyDictionary<string, DirectoryInfo> GetInstalledKernelDirectories();
```

Comment:
How does this relate to `GetDefaultKernelSpecDirectory`?

Reply:
The default is being used for installing the .NET kernel spec in this case whereas the list of installed kernel directories is returning all the directories where all the kernels are installed based on the search order specified here: https://docs.jupyter.org/en/latest/use/jupyter-directories.html where JUPYTER_PATH is searched first and then default.

Reply:
Would it make sense to consolidate them to ensure consistency?
.

Reply:
Jupyter seems to actually separate the concept here. There is a list of jupyter paths that it searches for when looking for kernels vs. a default data_dir that is either set or can be overridden via an environment variable JUPYTER_DATA_DIR. So it's aligned with jupyter and is designed to be used for two different scenarios.